extensions/bundle/README.md

```diff
@@ -33,16 +33,14 @@ bundle add my-package  # This will be checked by Phylum!
 
 ## How it works
 
-When invoking `phylum bundle`, subcommands that would install gems will trigger
-a Phylum analysis first.
-
-- If the analysis is successful, the corresponding changes will be applied.
-- If the analysis is unsuccessful because some of the new dependencies don't
-  meet the required project thresholds, the command will fail.
-- If the analysis is waiting for Phylum to process one or more of the submitted
-  packages, the command will fail and the changes will _not_ be applied.
-- Commands that modify neither `Gemfile`, nor `Gemfile.lock` will be passed
-  through to `bundle` directly.
+Wher running the package manager through this extension, subcommands which would
+install new packages will trigger a Phylum analysis first. Once that analysis
+passes Phylum's default policy, the installation is performed. If it did not
```

Comment:
A link to the [policy documentation](https://docs.phylum.io/docs/policy) would be nice here.

Reply:
Fair...but reading from a user's perspective, a natural question is going to be: "What is Phylum's default policy?" Heading that question off at the pass is the goal with this suggestion.

Reply:
I think the point of mentioning the default policy here is just to clarify for anyone that is using custom policies for their projects.

For users that are completely unfamiliar with our policy framework (or free users that cannot set custom policies), the default policy is all they know. A pass is a pass. A fail is a fail.

To adequately answer "What is Phylum's default policy?", you have to answer "What makes an issue high risk?"... It is unfortunately very subjective and I don't believe we even attempt to answer that question anywhere in our documentation.

Reply:
> ...I don't believe we even attempt to answer that question anywhere in our documentation.

Agreed...couldn't find it. Okay, leaving this unchanged is fine.